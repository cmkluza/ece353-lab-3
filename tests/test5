Tests ADDI without errors, with m = 1, n = 2, c = 1
./sim-mips -b 1 2 1 ./tests/asm/test1.s ./tests/out/test5.out
./tests/sols/test1.sol