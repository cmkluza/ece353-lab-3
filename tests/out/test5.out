program name: ./tests/asm/test1.s
stage utilization: 0.350000  0.300000  0.600000  0.000000  0.300000 
register values 0  0  0  0  0  0  0  15  10  0  0  0  0  0  0  20  7  0  0  0  0  0  0  0  0  0  0  0  5  0  8  28
